blog/meet-content-wro-luty-2024-relacja/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 'MeetContentWRO luty 2024 - Relacja'
+authors: mrozek
+date: '2024-03-14'
+tags:
+  - 'meetup'
+  - 'video produktowe'
+  - 'spolecznosc'
+coverImage: '25MeetContentWRO.png'
+---
+
+27 lutego spotkaliśmy się niestandardowo - w słynnej wrocławskiej klubokawiarni Mleczarnia, żeby posłuchać o tym, jak robi się filmiki produktowe. 
+
+Naszym gospodarzem tym razem była, skądinąd znana i umiejąca w filmiki, firma z branży IT - **CodeTwo**.
+
+<!--truncate-->
+
+Prezentację na temat: *Creating product videos - a team challenge* brawurowo i z humorem dostarczyli:  
+
+* [Michał Królewicz](https://www.linkedin.com/in/mkrolewicz/), który jest Technical Marketing Managerem, 
+* [Kacper Tomaszewski](https://www.linkedin.com/in/kacpertomaszewski/), który jest Senior Graphic & UI/UX Designerem, oraz 
+* [Kamil Glaser](https://www.linkedin.com/in/kamil-glaser-92b91482/), który jest Senior Technical Marketing Specialist
+
+# Po co tworzyć filmy produktowe?
```

Comment:
```suggestion
## Po co tworzyć filmy produktowe?
```

H1 to tytuł całego artykułu, więc nagłówki w samym artykule powinny się zaczynać od h2. Czy możesz zmienić wszystkie, żeby były o poziom niżej. 

To ważne trochę z powodu layoutu i stylowania, ale bardziej z powodu accessibility - screen reader działa lepiej, jeśli hierarchia nagłówków jest zachowana.

Reply:
@pensjonatus  Ok, przepraszam, myślałam, że jak zrobię Dismiss review to po prostu zlikwiduje komentarz... A on zlikwidował apporval... ;) 
Dasz jeszcze raz approva? Dziękuję!